vmm/src/api/mod.rs

```diff
@@ -258,6 +258,12 @@ pub struct VmSendMigrationData {
     /// Send memory across socket without copying
     #[serde(default)]
     pub local: bool,
+    /// Downtime
+    #[serde(default)]
+    pub downtime: u64,
```

Comment:
```suggestion
    pub downtime: Duration,
```

Reply:
This would the code much clearer and people don't need to remember "is this second? Is this minute? Is this milliseconds?"